backend/services/itinerary-service/internal/db/itinerary-db.go

```diff
@@ -0,0 +1,222 @@
+package DBmodels
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	"github.com/lib/pq"
+)
+
+type Itinerary struct {
+	ItineraryId int      `json:"itineraryId"`
+	Name        string   `json:"name"`
+	City        string   `json:"city"`
+	Country     string   `json:"country"`
+	Title       string   `json:"title"`
+	Description string   `json:"description"`
+	Price       float64  `json:"price"`
+	Languages   []string `json:"languages"`
```

Comment:
I don't know if we need to have languages in itineraries I feel like if there are any language barriers the user could just put it in the description.

Reply:
I don't have any stronger view on this tbh, and am good with any implementation. I'll ask the rest of the team.